project/home/main.py

```diff
@@ -6,5 +6,14 @@
 # 입력: 유저의 승부 이력
 # 출력: 컴퓨터가 가위, 바위, 보 중 어떤 것을 선택할 것인지 결정 후 반환
 def logic(history):
-    choices = [item[0] for item in CHOICES]
-    return random.choice(choices)
+    #유저가 마지막으로 냈던 것
```

Comment:
보통, # 다음에는 스페이스를 하나 둡니다. 
```suggestion
    # 유저가 마지막으로 냈던 것
```